optimade/validator/validator.py

```diff
@@ -410,7 +414,12 @@ def _test_must_properties(
             `True` if the properties were found, and a string summary.
 
         """
-        must_props = CONF.entry_properties[endp]["MUST"]
+        must_props = set(
+            field
+            for field in CONF.entry_schemas[endp]
+            if CONF.entry_schemas[endp].get(field, {}).get("support")
```

Comment:
Just to be a bit consistent with the variable naming.
```suggestion
            prop
            for prop in CONF.entry_schemas[endp]
            if CONF.entry_schemas[endp].get(prop, {}).get("support")
```